tests/test_utils.py

```diff
@@ -5,8 +5,8 @@
 def test_json_loader_testfile():
     """Test creating spectrum(s) from test json file."""
     path_tests = os.path.dirname(__file__)
-    testfile = os.path.join(path_tests, "testspectrum.json")
```

Comment:
As far as I see ``testspectrum.json`` is not needed anymore. If so, just delete it.

Reply:
Yes indeed. I'll do that.